systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java

```diff
@@ -467,6 +511,23 @@ && cmdKubeClient().namespace(testStorage.getNamespaceName()).execInPodContainer(
     @ParallelNamespaceTest
     @Tag(BRIDGE)
     @Tag(ROLLING_UPDATE)
+    @TestDoc(
+        description = @Desc("This test dynamically changes the logging levels of the Kafka Bridge and verifies the application behaves correctly with respect to these changes."),
+        steps = {
+            @Step(value = "Configure initial logging levels to OFF.", expected = "Logging levels are set to OFF."),
+            @Step(value = "Deploy Kafka resources asynchronously and wait for their readiness.", expected = "Kafka resources, including the Bridge, become ready."),
```

Comment:
I would not say "Kafka resources" when this test checks the Bridge logging. It's a bit confusing when you are reading through it. Also, you are configuring this only in the Bridge, so I would write the steps differently.